src/runtime/graph/debug/graph_runtime_debug.cc

```diff
@@ -110,6 +110,16 @@ class GraphRuntimeDebug : public GraphRuntime {
   }
 
   double RunOpRPC(int index, int number, int repeat, int min_repeat_ms) {
+    if (nodes_[index].op_type != "tvm_op") {
```

Comment:
Could you just leave a comment for why we are doing this

Reply:
done